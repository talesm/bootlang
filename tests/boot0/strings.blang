writeln('hello'.concat(' world!'));
writeln('hello'.slice(1, 3));
writeln('hi'.getLength().toString());
writeln('hi'.equals('hi').toString());
writeln('hi'.equals('hi!').toString());
writeln('hi'.isBefore('ha').toString());
writeln('hi'.isBefore('hii').toString());
writeln('hi'.isAfter('ha').toString());
writeln('hi'.isAfter('hii').toString());
writeln(' '.toAscii().toString());
writeln(string.fromAscii(32));
