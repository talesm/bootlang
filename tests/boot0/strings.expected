hello world!
ell
2
true
false
false
true
true
false
32
 
